textproc/grc/Portfile

```diff
@@ -25,7 +24,7 @@ checksums           rmd160  06578c83ffae331825fc1b5fe8ac1b5279f1f44a \
                     sha256  1468c50237fc260840384cc2063f3031689e95e5c6dc22533aa5d25a3d17f2f9 \
                     size    49236
 
-python.default_version 39
+python.default_version 311
```

Comment:
changing `python.default_version` will result in a different `depends_lib` dependency (i.e., `python311` instead of `python39`) so that requires a `revisions` increase